src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,81 @@
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+import { sortPeople } from '../../utils/sortPeople';
+import { SearchLink } from '../SearchLink';
+import PersonLink from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const [searchParams] = useSearchParams();
+
+  const getSortIcon = (field: string | null) => {
+    const sortField = searchParams.get('sort' || null);
+    const sortOrder = searchParams.get('order' || null);
+
+    if (sortField === field) {
+      return sortOrder === null ? 'fa-sort-up' : 'fa-sort-down';
+    }
+
+    return 'fa-sort';
+  };
+
+  const toggleSortParams = (field: string) => {
+    const currentSort = searchParams.get('sort');
+    const currentOrder = searchParams.get('order');
```

Comment:
You get these values from params in `getSortIcon` too, move these variables from functions so they both can use them